couch/assets/dashboards/CouchDB-overview_dashboard.json

```diff
@@ -0,0 +1,837 @@
+            "layout": {
+                "height": 14,
+                "width": 32,
+                "x": 1,
+                "y": 1
+            }
+        },
+        {
+            "definition": {
+                "background_color": "vivid_blue",
+                "content": "Performance",
+                "font_size": "18",
+                "show_tick": false,
+                "text_align": "center",
+                "tick_edge": "bottom",
+                "tick_pos": "50%",
+                "type": "note"
+            },
+            "id": 16,
+            "layout": {
+                "height": 5,
+                "width": 42,
+                "x": 121,
+                "y": 1
+            }
+        },
+        {
+            "definition": {
+                "background_color": "blue",
+                "content": "Traffic Breakdown",
+                "font_size": "18",
+                "show_tick": false,
+                "text_align": "center",
+                "tick_edge": "bottom",
+                "tick_pos": "50%",
+                "type": "note"
+            },
+            "id": 17,
+            "layout": {
+                "height": 5,
+                "width": 85,
+                "x": 35,
+                "y": 68
+            }
+        },
+        {
+            "definition": {
+                "background_color": "vivid_blue",
+                "content": "Authentication Cache",
+                "font_size": "18",
+                "show_tick": false,
+                "text_align": "center",
+                "tick_edge": "bottom",
+                "tick_pos": "50%",
+                "type": "note"
+            },
+            "id": 18,
+            "layout": {
+                "height": 5,
+                "width": 42,
+                "x": 121,
+                "y": 39
+            }
+        },
+        {
+            "definition": {
+                "custom_links": [],
+                "legend_size": "0",
+                "markers": [
+                    {
+                        "display_type": "error dashed",
+                        "value": "y = 3"
+                    }
+                ],
+                "requests": [
+                    {
+                        "display_type": "line",
+                        "q": "avg:couchdb.couchdb.httpd_status_codes.401{$couchdb,$couchdb,$scope}",
+                        "style": {
+                            "palette": "red"
+                        }
+                    },
+                    {
+                        "display_type": "bars",
+                        "q": "avg:couchdb.couchdb.auth_cache_misses{$couchdb,$scope}",
+                        "style": {
+                            "palette": "orange"
+                        }
+                    }
+                ],
+                "show_legend": false,
+                "time": {
+                    "live_span": "4h"
+                },
+                "title": "Unauthorized login attempt",
```

Comment:
should this be "attempts"?

Reply:
Maybe. I only updated the metric names and the rest were kept the same. I'll update it for both.